src/components/EditCoursePage/index.jsx

```diff
@@ -433,6 +439,7 @@ class EditCoursePage extends React.Component {
                 owners={owners}
                 isSubmittingForReview={isSubmittingForReview}
                 targetRun={targetRun}
+                editable={editable}
```

Comment:
Thoughts on passing in a `disabled` prop that you define here rather than have to redefine `disabled = courseInReview | ~editable` in the edit form and collapsible course runs.

Reply:
I did it this way because the CollapsibleCourseRun wants to know editable status standalone (so it knows whether to show the submit button). So it is interested in both fields separately. I guess I could pass in `disabled` *plus* `editable`, so it doesn't have to calculate it... But then it feels weird that it ignores the `disabled` flag for the submit button sometimes. If it decides what is disabled, it feels less odd that it ignores orders for that one button.